hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/policy/ContainerChoosingPolicy.java

```diff
@@ -28,9 +28,14 @@
 public interface ContainerChoosingPolicy {
   /**
    * Choose a container for balancing.
-   *
+   * @param inProgressContainerIDs containerIDs present in this set should be
```

Comment:
please add @param for ozoneContainer and volume too, to complete the jave doc for this function.